package.json

```diff
@@ -10,6 +10,7 @@
   },
   "dependencies": {
     "firebase": "^11.4.0",
+    "framer-motion": "^12.5.0",
```

Comment:
Why?

Framer Motion seems _very_ overkill for such a small project, and you didn’t seem to have used it anyway, so consider uninstalling it?